### PR TITLE
Add HEOS media player component docs

### PR DESCRIPTION
**Pull request by easink**
**Description:**

Denon HEOS speaker.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21721

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/media_player.heos.page
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]: missing the default

File: source/_components/media_player.heos.page
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]: You can remove this line

File: source/_components/media_player.heos.page
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]:
```suggestion
    host: IP_ADDRESS
```

File: source/_components/media_player.heos.page
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]: missing `ha_release` and a logo

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/media_player.heos.page
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]:
```suggestion
ha_release: 0.91
```

File: source/_components/media_player.heos.page
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]:
```suggestion
  description: "Address of the device. Example: 192.168.1.32."
```

File: source/_components/media_player.heos.page
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]: Are you using this link? I see no reference to it

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/media_player.heos.page
Comment (easink) [reply source/_components/media_player.heos.page]: nope, removed

**Review by klaasnicolaas — APPROVED** [review]
✅ Can be merged after merging the parent PR 👍

**Pull request merged**
